android/demo/src/main/java/com/intuit/playerui/android/reference/demo/lifecycle/DemoPlayerViewModel.kt

```diff
@@ -24,8 +24,10 @@ class DemoPlayerViewModel(iterator: AsyncFlowIterator) : PlayerViewModel(iterato
         AsyncHydrationTrackerPlugin(),
     )
 
+    public val isDebug = false
```

Comment:
Nit: another source of truth for `config.debuggable`

Reply:
it is, it's purely for demo because of [this](https://github.com/player-ui/player/pull/443/files#diff-9ee98332d0ca1039bddbbec92d72d2f5cf5b0c4feba501c358b9dca1f23d7c75R33) check, otherwise i think it complains about classes not being available on classpath